src/components/reference/_triage-states.mdx

```diff
@@ -0,0 +1,11 @@
+| Status | Description |
+| -----------  | ------------ |
+| **Open** | Findings are open by default. A finding is open if it was present the last time Semgrep scanned the code and it has not been ignored. An open finding represents a match between the code and a rule that is enabled in the repository. Open findings require action, such as rewriting the code to eliminate the detected vulnerability. |
+| **Ignored** | Findings that are ignored are present in the code, but have been labeled as unimportant. Ignore findings that are false positives or deprioritized issues. Mark findings as ignored through Semgrep App (see [Managing finding status](/semgrep-app/findings/#managing-finding-status-bulk-triage)) or by adding a code comment (see [nosemgrep comments](/ignoring-files-folders-code/#reference-summary)). |
+| **Fixed** | <p>Fixed findings were detected in a previous scan, but are no longer detected in the most recent scan of that same branch. </p> <p> To change the state of a finding to **fixed**: <ol type="a"><li>Fix the code so the rule does not match it.</li><li>Edit the Semgrep rule so it no longer matches the code.</li></ol></p> |
```

Comment:
@chmccreery why did we remove `The Semgrep rule that matched the finding and the code that triggered the match must both be active in the most recent scan.` Is this no longer correct?

Reply:
It is correct, but it felt a bit clunky - I think that the definition of "removed" below makes this somewhat self-evident. If you disagree, I'm happy to add it back!